dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -4179,7 +4179,7 @@ paths:
                 type: boolean
           description: default response
     get:
-      operationId: getUser
+      operationId: getUser_1
```

Comment:
This op is still swapping back and forth with the following. 
Try to set the operationID properly in a follow-up ticket.